notebooks/10_clustering_and_regionalization.md

```diff
@@ -270,10 +218,12 @@ numpy.random.seed(123456)
 # Calculate Moran's I for each variable
 mi_results = [Moran(db[variable], w) for variable in cluster_variables]
 # Display on table
-table = pandas.DataFrame([(variable, res.I, res.p_sim) for variable,res 
-                      in zip(cluster_variables, mi_results)],
-                     columns=['Variable', "Moran's I", 'P-value'])\
-          .set_index('Variable')
+table = pandas.DataFrame([(variable, res.I, res.p_sim) \
+                          for variable,res \
```

Comment:
line continuation characters shouldn't be necessary until the step at 225, since they're within a parenthesis for `DataFrame`... by they're fine in for now.